lib/cloud/aws/policy_statements.go

```diff
@@ -160,6 +160,7 @@ func StatementForListRDSDatabases() *Statement {
 		Actions: []string{
 			"rds:DescribeDBInstances",
 			"rds:DescribeDBClusters",
+			"ec2:DescribeSecurityGroups",
```

Comment:
testing it on the CLI running:

```
teleport integration configure listdatabases-iam --aws-region=us-east-1 --role=lk-role-name-3
```
<img width="488" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/1216997f-3467-4514-8050-0189f84d2da1">